worker/uniter/runner/context/export_test.go

```diff
@@ -102,9 +102,10 @@ func NewMockUnitHookContext(mockUnit *mocks.MockHookUnit) *HookContext {
 
 func NewMockUnitHookContextWithState(mockUnit *mocks.MockHookUnit, state *uniter.State) *HookContext {
 	return &HookContext{
-		unit:   mockUnit,
-		state:  state,
-		logger: loggo.GetLogger("test"),
+		unitName: mockUnit.Tag().Id(),
```

Comment:
Perhaps a comment outlining why we need or want a unitName?

Reply:
yup